Anytype/Sources/PresentationLayer/Modules/SpaceSwitch/Components/SpaceRowView.swift

```diff
@@ -12,27 +12,38 @@ struct SpaceRowModel: Identifiable {
 
 struct SpaceRowView: View {
     
+    private enum Constants {
+        static let lineWidth: CGFloat = 3
+    }
+    
     static let width: CGFloat = 96
     
     let model: SpaceRowModel
     
     var body: some View {
         VStack {
-            IconView(icon: model.icon)
-                .frame(width: Self.width, height: 96)
-                .cornerRadius(8)
-                .shadow(color: .Shadow.primary, radius: 20)
-                .if(model.isSelected) {
-                    $0.border(8, color: .Text.white, lineWidth: 3)
+            ZStack {
+                ZStack {
+                    // Outer border
+                    if model.isSelected {
+                        Color.Text.white
+                            .cornerRadius(4)
+                    }
+                    IconView(icon: model.icon)
+                        .frame(width: Self.width, height: Self.width)
+                        .shadow(color: .Shadow.primary, radius: 20)
                 }
```

Comment:
I tried and received a error with contextMenu, because contextMenu show view without out out bounds view. Inner ZStack is view with bigger bounds for show context menu with white border

Reply:
👌